### PR TITLE
ACCUMULO-4503 Added blog to project website

### PR DESCRIPTION
**Pull request by mikewalch**
* Upgraded bootstrap
* Changed website theme
* Redesigned home page

### CODE REVIEW

**Comment by keith-turner** [comment]
For the blog post, could include a link to the original post along with a sentence stating it was copied.   This could be at the end of the post.

**Comment by joshelser** [comment]
> For the blog post, could include a link to the original post along with a sentence stating it was copied. This could be at the end of the post. 

+1

Not related to this changeset, but we should also make a post on blogs.a.o saying that it is discontinued, pointing people back to accumulo.a.o/blog

**Comment by ctubbsii** [comment]
All looks good to me. I'll let this sit overnight for additional reviews, then merge tomorrow. Thanks, @mikewalch !

**Comment by joshelser** [comment]
> I'll let this sit overnight for additional reviews, then merge tomorrow

Deployed locally and saw all posts are called the "news archive". Seems confusing to me to use both the term "blog" and "news". Also, I would have expected to find the archives at "/blog", but that is just an directory listing. The archives are actually at "/news". This is a little nit-picky, I know. If it's something simple that can be changed, I think it would be beneficial. However, if it's going to go against the grain of what Jekyll wants you to do, I'm not going to get upset over it.

**Comment by keith-turner** [comment]
> This is a little nit-picky, I know. If it's something simple that can be changed, I think it would be beneficial.

w.r.t. to blog post links it would be nice to get something settled that will be stable, so the links will be stable over time.

**Comment by ctubbsii** [comment]
@joshelser That looked funny to me, too, until I realized that News activity could include other kinds of posts than blogs, such as the release posts that Apache Fluo is doing.

We can add an additional archive of just the blogs at the root of /blog. That's no big deal.

As for any potential releases feed, we might want to think about doing that as follow-on work later, after we folks get a chance to acclimate to this change and provide feedback.

**Comment by mikewalch** [comment]
@joshelser, with this pull request, I am trying to mimic what has been done on the Apache Fluo website.  The Fluo website has two kinds of Jekyll posts (blog and release). Blog posts are self-explanatory. Release posts contain the release notes for the release and links to resources (binary tarball, docs, javadocs, etc) for that release. Below are examples each post type:

https://fluo.apache.org/blog/2016/06/02/fluo-moving-to-apache/
https://fluo.apache.org/release/fluo-1.0.0-incubating/

When you go to https://fluo.apache.org/news/, you see all posts (blog & release).  This was done to avoid creating two posts (a release post and a blog post announcing the release).  When you go https://fluo.apache.org/release/, you only see releases (the pre-apache releases have been excluded).  Currently, the page https://fluo.apache.org/blog/ redirects to /news/ but it could show only blog posts.  This system has made releases easier to Fluo.  You just need to create one markdown document and the release will show up in 3 locations automatically ( 'latest news' on main page, the news archive '/news/', and release archive '/release/').

**Comment by mikewalch** [comment]
To be clear, this PR doesn't make releases Jekyll posts but I would like to do this in a later PR.

**Comment by joshelser** [comment]
>  The Fluo website has two kinds of Jekyll posts (blog and release). Blog posts are self-explanatory. Release posts contain the release notes for the release and links to resources (binary tarball, docs, javadocs, etc) for that release.

Aha, this makes sense now. Thanks for the explanation.

>  Currently, the page https://fluo.apache.org/blog/ redirects to /news/ but it could show only blog posts.

That satisfies me!

> To be clear, this PR doesn't make releases Jekyll posts but I would like to do this in a later PR.

+1

File: css/accumulo.css [hunk 1]
```diff
@@ -15,37 +15,67 @@
...
-    font-size: 14px;
+  font-size: 14px;
 }
 
-.nav-link {
-    padding-top: 20px;
-    font-size: 16px;
+code {
+  background-color: #f5f5f5;
+  color: #555;
```
Comment (ctubbsii) [inline css/accumulo.css]: Suggest a greenish color here like `#20b2aa;`

**Pull request merged**